sdk/python/feast/transformation/substrait_transformation.py

```diff
@@ -16,14 +18,16 @@
 
 
 class SubstraitTransformation:
-    def __init__(self, substrait_plan: bytes):
+    def __init__(self, substrait_plan: bytes, ibis_function: FunctionType):
```

Comment:
should this param be optional?

Reply:
I think it should be fine this way. If we find some way to make it totally independent from ibis, then we can make it optional...